gtsfm/utils/geometry_comparisons.py

```diff
@@ -30,138 +30,234 @@ def align_rotations(input_list: List[Rot3], ref_list: List[Rot3]) -> List[Rot3]:
     return [w2Rw1.compose(w1Ri) for w1Ri in input_list]
 
 
-def align_poses(input_list: List[Pose3], ref_list: List[Pose3]) -> List[Pose3]:
-    """Aligns the list of poses to the reference list by shifting origin and
-    scaling translations.
+def align_translations(input_list: List[Point3], ref_list: List[Point3]) -> Tuple[List[Point3], float, Rot3, Point3]:
```

Comment:
i think the alignment stuff should go in a separate PR, since it's still a WIP.

I propose we add the unit tests for now, and check in commented-out tests if they wont pass for now. Then we slowly try to uncomment them in future PRs